up42/workflow.py

```diff
@@ -342,6 +342,43 @@ def construct_parameters(
             input_parameters[data_block_name][geometry_operation] = aoi_feature
         return input_parameters
 
+    def mapping_to_parameters(
+        self,
+        geometries: List[Union[Dict, Feature, geojson_Polygon, Polygon, Point,]],
+        interval_dates: List[Tuple[str, str]],
```

Comment:
Would it be better to start with `image_ids` instead of `interval_dates` here to make it easier to grasp? Think this was suggested in the discussion. Because date handling in the sdk is still lacking (missing `time_series` in `construct_parameters`, accepted date formats should be extended) this would also keep our options open to  then just add `time_series` here.

Reply:
The scope for this was pretty clear in the point were we want to support multiple geoms and multiple timestamps  i.e. 
```
1. Same AOI, multiple timesteps
- limit N → N datasets → N jobs
2. Different AOIs, same/different timesteps 
- N AOIs → N datasets → N Jobs
```
Are you hinting that the user would always then do the catalog search and then run this mapping?

Reply:
I also thought the same. Since this will affect the parallelization as well, I guess we need to be clear what exactly we want.

Reply:
A okay, think we misscommunicated. Thought about timesteps as multiple scenes, did not consider the time periods we use via `time_series` parameter. @markusuwe also suggested the catalog output scene_ids as the input I think, maybe I missed the conclusion. How would parallelizing multiple scene_ids be handled, would that need to be added here?

Would `construct_parameters_parallel` a more descriptive name?

Reply:
Changed name of method and added three cases as discussed.